assets/js/stripe.js

```diff
@@ -268,6 +260,16 @@ jQuery( function( $ ) {
 					return false;
 				} );
 			}
+
+			// Listen for hash changes in order to handle payment intents
+			window.addEventListener( 'hashchange', wc_stripe_form.onHashChange );
+			if ( $( '.stripe-intent-id' ).length && $( '.stripe-intent-return' ).length ) {
```

Comment:
Is there any chance there could be multiple elements with this class present in the DOM?

Reply:
Both of those names (`stripe-intent-id` and `stripe-intent-return`) are unique for this situation and would not be displayed multiple times on the same page.

Reply:
So, why classes and not IDs then?

Reply:
I was merely following the example of all other methods in the file. I don't mind though, so I switched to IDs.